lib/amazon/parser/flavor.rb

```diff
@@ -48,8 +48,9 @@ def parse_flavor_storage(storage)
 
       def parse_flavor_memory(memory)
         match = /^((\d*[.])?(\d+)).*$/.match(memory&.gsub(",", ""))
-        return 0 unless match
-        (match[1].to_f * 1024 ** 3).to_i # convert GiB to B
+        return unless match
+
+        (match[1].to_f * 1024**3).to_i # convert GiB to B
```

Comment:
We could just include `active_support/core_ext/numeric/bytes` and use `1.gigabyte` here